### PR TITLE
dev,3.0:update alibaba cloud deployment document

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

### What is changed, added or deleted? 
update alibaba cloud deployment document

### What is the related PR or file link(s)? 
https://github.com/pingcap/tidb-operator/pull/658

### Which version does your change affect? 
`dev`, `3.0`

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 1]
```diff
@@ -156,23 +154,147 @@ $ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
 > 监控组件挂载的云盘需要在阿里云管理控制台中手动删除。
 
-## 自定义集群
+## 配置
 
-默认配置下，Terraform 脚本会创建一个新的 VPC，假如要使用现有的 VPC，可以在 `variable.tf` 中设置 `vpc_id`。注意，当使用现有 VPC 时，没有设置 vswitch 的可用区将不会部署 Kubernetes 节点。
+### 配置 TiDB Operator
 
-出于安全考虑，TiDB 服务的 SLB 只对内网暴露，因此默认配置下还会创建一台堡垒机用于运维操作。堡垒机上还会安装 mysql-cli 和 sysbench 以便于使用和测试。假如不需要堡垒机，可以设置 `variables.tf` 中的 `create_bastion` 参数来关闭。
+通过调整 `variables.tf` 内的值来配置 TiDB Operator，大多数配置项均能按照 `variable` 的注释理解语义后进行修改。需要注意的是 `operator_helm_values` 配置项，该配置项允许为 TiDB Operator 提供一个自定义的 `values.yaml` 配置文件，示例如下：
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
通过调整 `variables.tf` 内的值来配置 TiDB Operator，大多数配置项均能按照 `variable` 的注释理解语义后进行修改。需要注意的是，`operator_helm_values` 配置项允许为 TiDB Operator 提供一个自定义的 `values.yaml` 配置文件，示例如下：
```

File: dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 2]
```diff
@@ -156,23 +154,147 @@ $ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
+
+1. 首先针对每个集群创建一个目录，如：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    mkdir -p deploy/aliyun/aliyun-staging
+    ```
+
+2. 参考 `deploy/aliyun` 的 `main.tf`，编写自己的脚本，下面是一个最简单的例子：
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
2. 参考 `deploy/aliyun` 的 `main.tf`，编写自己的脚本，下面是一个简单的例子：
```

File: dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 3]
```diff
@@ -156,23 +154,147 @@ $ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
+        enable_ssh_to_worker     = true
+        worker_security_group_id = module.tidb-operator.security_group_id
+    }
+    ```
 
-特殊地，由于 PD 和 TiKV 节点强需求本地 SSD 存储，脚本中不允许直接声明 PD 和 TiKV 的规格名，你可以通过设置 `*_instance_type_family` 来选择 PD 或 TiKV 的规格族（只能在三个拥有本地 SSD 的规格族中选择），再通过内存大小来筛选符合需求的型号。
+上面的脚本可以自由定制，比如，假如不需要堡垒机则可以移除 `module "bastion"` 相关声明。
 
-更多自定义配置相关的内容，请直接参考项目中的 `variables.tf` 文件。
+你也可以直接拷贝 `deploy/aliyun` 目录，但要注意不能拷贝已经运行了 `terraform apply` 的目录，最好的办法是重新 clone 仓库再进行拷贝。
```
Comment (lilin90) [inline dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
你也可以直接拷贝 `deploy/aliyun` 目录，但要注意不能拷贝已经运行了 `terraform apply` 的目录，建议重新 clone 仓库再进行拷贝。
```

File: v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 4]
```diff
@@ -157,23 +154,147 @@ $ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
 > 监控组件挂载的云盘需要在阿里云管理控制台中手动删除。
 
-## 自定义集群
+## 配置
 
-默认配置下，Terraform 脚本会创建一个新的 VPC，假如要使用现有的 VPC，可以在 `variable.tf` 中设置 `vpc_id`。注意，当使用现有 VPC 时，没有设置 vswitch 的可用区将不会部署 Kubernetes 节点。
+### 配置 TiDB Operator
 
-出于安全考虑，TiDB 服务的 SLB 只对内网暴露，因此默认配置下还会创建一台堡垒机用于运维操作。堡垒机上还会安装 mysql-cli 和 sysbench 以便于使用和测试。假如不需要堡垒机，可以设置 `variables.tf` 中的 `create_bastion` 参数来关闭。
+通过调整 `variables.tf` 内的值来配置 TiDB Operator，大多数配置项均能按照 `variable` 的注释理解语义后进行修改。需要注意的是 `operator_helm_values` 配置项，该配置项允许为 TiDB Operator 提供一个自定义的 `values.yaml` 配置文件，示例如下：
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
通过调整 `variables.tf` 内的值来配置 TiDB Operator，大多数配置项均能按照 `variable` 的注释理解语义后进行修改。需要注意的是，`operator_helm_values` 配置项允许为 TiDB Operator 提供一个自定义的 `values.yaml` 配置文件，示例如下：
```

File: v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 5]
```diff
@@ -157,23 +154,147 @@ $ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
+
+1. 首先针对每个集群创建一个目录，如：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    mkdir -p deploy/aliyun/aliyun-staging
+    ```
+
+2. 参考 `deploy/aliyun` 的 `main.tf`，编写自己的脚本，下面是一个最简单的例子：
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
2. 参考 `deploy/aliyun` 的 `main.tf`，编写自己的脚本，下面是一个简单的例子：
```

File: v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md [hunk 6]
```diff
@@ -157,23 +154,147 @@ $ terraform state rm module.ack.alicloud_cs_managed_kubernetes.k8s
...
+        enable_ssh_to_worker     = true
+        worker_security_group_id = module.tidb-operator.security_group_id
+    }
+    ```
 
-特殊地，由于 PD 和 TiKV 节点强需求本地 SSD 存储，脚本中不允许直接声明 PD 和 TiKV 的规格名，你可以通过设置 `*_instance_type_family` 来选择 PD 或 TiKV 的规格族（只能在三个拥有本地 SSD 的规格族中选择），再通过内存大小来筛选符合需求的型号。
+上面的脚本可以自由定制，比如，假如不需要堡垒机则可以移除 `module "bastion"` 相关声明。
 
-更多自定义配置相关的内容，请直接参考项目中的 `variables.tf` 文件。
+你也可以直接拷贝 `deploy/aliyun` 目录，但要注意不能拷贝已经运行了 `terraform apply` 的目录，最好的办法是重新 clone 仓库再进行拷贝。
```
Comment (lilin90) [inline v3.0/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md]:
```suggestion
你也可以直接拷贝 `deploy/aliyun` 目录，但要注意不能拷贝已经运行了 `terraform apply` 的目录，建议重新 clone 仓库再进行拷贝。
```

**Comment by aylei** [comment]
@lilin90 Thanks for the careful review!

**Pull request merged**